### PR TITLE
Scripts to create the RPM for the Context Adapter

### PR DESCRIPTION
**Pull request by dmoranj**
ADD Specs and scripts to generate RPM files in order to install the Context Adapter.

### CODE REVIEW

File: rpm/SOURCES/etc/cron.d/cron-logrotate-contextAdapter-size [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of the iotagent-context-adapter.
```
Comment (gtorodelvalle) [inline rpm/SOURCES/etc/cron.d/cron-logrotate-contextAdapter-size]: I saw some references to `iotagent-context-adapter`in this file... I guess you mean `context-adapter` right? :)

File: rpm/SOURCES/etc/cron.d/cron-logrotate-contextAdapter-size [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with the iotagent-context-adapter. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# cron job to test and execute log rotate by size.
+*/30 * * * * iotatt /usr/sbin/logrotate /etc/sysconfig/logrotate-contextAdapter-size
```
Comment (gtorodelvalle) [inline rpm/SOURCES/etc/cron.d/cron-logrotate-contextAdapter-size]: Is this line right? I mean regarding the `iotatt` :)

File: rpm/SOURCES/etc/init.d/contextAdapter [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    'restart')
+        ca_stop
+        ca_start
+        ;;
+
+    'status')
+        ca_status
+        ;;
+
+esac
```
Comment (gtorodelvalle) [inline rpm/SOURCES/etc/init.d/contextAdapter]: Final blank line appreciated :)

File: rpm/SOURCES/etc/logrotate.d/logrotate-contextAdapter.conf [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of the iotagent-thinking-things.
```
Comment (gtorodelvalle) [inline rpm/SOURCES/etc/logrotate.d/logrotate-contextAdapter.conf]: I saw some references to `iotagent-context-adapter` in this file... I guess you mean `context-adapter` right? :)

File: rpm/SOURCES/etc/sysconfig/logrotate-contextAdapter-size [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of the iotagent-thinking-things.
```
Comment (gtorodelvalle) [inline rpm/SOURCES/etc/sysconfig/logrotate-contextAdapter-size]: I saw some references to `iotagent-context-adapter` in this file... I guess you mean `context-adapter` right? :)

File: rpm/create-rpm.sh [hunk 6]
```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of the iotagent-thinking-things.
```
Comment (gtorodelvalle) [inline rpm/create-rpm.sh]: I saw some references to `iotagent-context-adapter` in this file... I guess you mean `context-adapter` right? :)

File: rpm/create-rpm.sh [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+if [[ ! -z ${RELEASE_ARG} ]]; then
+	PRODUCT_RELEASE=${RELEASE_ARG}
+else
+	echo "A product reslease must be specified with -r parameter."
+	usage
+	exit 2
+fi
+
+PROXY_USER="iotatt"
```
Comment (gtorodelvalle) [inline rpm/create-rpm.sh]: Is this right? I mean the reference to `iotatt`:)

File: rpm/SOURCES/etc/cron.d/cron-logrotate-contextAdapter-size (hunk 1 above)
Comment (dmoranj) [inline rpm/SOURCES/etc/cron.d/cron-logrotate-contextAdapter-size]: Yes, I meant that. I'll change it.

File: rpm/SOURCES/etc/cron.d/cron-logrotate-contextAdapter-size (hunk 2 above)
Comment (dmoranj) [inline rpm/SOURCES/etc/cron.d/cron-logrotate-contextAdapter-size]: No, it's not. Changing.

**Comment by gtorodelvalle** [comment]
LGTM and merging!!! :)

**Pull request merged**